docs/GettingStartedDocs/Contributors/SGX1FLCGettingStarted.md

```diff
@@ -98,7 +95,7 @@ Test project /home/youradminusername/openenclave/build
 Total Test time (real) =  83.61 sec
 ```
 
-A clean pass of the above unitests run is an indication that your Open Enclave setup was successful. You can start playing with the Open Enclave samples after following the instructions in the "Install" section below to configure samples for building,
+A clean pass of the above unit tests run is an indication that your Open Enclave setup was successful. You can start playing with the Open Enclave samples after following the instructions in the "Install" section below to configure samples for building,
```

Comment:
"unit tests run" I believe this to be incorrect English as the verb is after the identifier. 

I would recommend changing to:

"A clean pass of the above unit tests is an indication of.."
or
"A clean run and passing of the above unit tests is an indication of.."

To remove potential ambiguity.

Reply:
Fixed.